src/pages/gen1/[platform]/prev/build-a-backend/auth/existing-resources-no-cli/index.mdx

```diff
@@ -0,0 +1,37 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Use existing resources without the CLI',
+  description:
+    'Configure the Amplify Libraries to use existing Amazon Cognito resources by referencing them in your configuration.',
+  platforms: ['flutter']
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+import flutter_maintenance from '/src/fragments/lib-v1/flutter-maintenance.mdx';
+
+<Fragments fragments={{ flutter: flutter_maintenance }} />
+
+import android0 from '/src/fragments/lib/auth/existing-resources.mdx';
+
+<Fragments fragments={{ android: android0 }} />
+
+import ios1 from '/src/fragments/lib/auth/existing-resources.mdx';
+
+<Fragments fragments={{ swift: ios1 }} />
```

Comment:
did you mean to add in all these android/ios bits and pieces?

Reply:
This was transferred from the previous V1 docs so I kept those swift and android fragments in there and just removed them from the export so that when swift release a new major version, they could just change the export to `platforms: ['flutter', 'swift', 'android']` instead of having to import all the fragments again.